filament/backend/src/vulkan/caching/VulkanPipelineLayoutCache.h

```diff
@@ -36,8 +36,8 @@ class VulkanPipelineLayoutCache {
     void terminate() noexcept;
 
     struct PushConstantKey {
-        uint8_t stage;// We have one set of push constant per shader stage (fragment, vertex, etc).
-        uint8_t size;
+        uint8_t stage = 0;// We have one set of push constant per shader stage (fragment, vertex, etc).
+        uint8_t size = 0;
```

Comment:
couldn't the problem come from that?

Reply:
that's what i thought. But just making this change does not fix the problem unfortunately.